### PR TITLE
bug/DL-63

### PR DESCRIPTION
**Pull request by wojwarych**
1. Any files concerning (more or less) Docker are now under `docker/` directory
2. Moved scripts concerning clearing docker and cache to the same directory
3. Added `.dockerignore` file so docker doesn't export unwanted files to container
3. Removed unnecessary entries in `requirements.txt` (some of them are just installed through dependencies)
4. Modified `.travis.yml` file so it gets docker .yml file from `docker/` dir
5. Added scripts for UNIX and Windows in order to run docker-compose from main directory of repo
6. Also added extra bash script just for building development image of docker for testing purposes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,20 +4,53 @@ Melvil application for Tieto Python Incubator.
```
Comment (kuszcjan) [inline README.md]: I would put info here how to run tests.

File: docker/docker-compose-dev.yml [hunk 2]
```diff
@@ -2,10 +2,12 @@ version: '3'
...
   web:
-    build: .
-    command: sh run.sh
+    build:
+      context: ..
+      dockerfile: './docker/Dockerfile'
+    command: sh './docker/run.sh'
     volumes:
-      - .:/code
+      - ..:/code
```
Comment (kuszcjan) [inline docker/docker-compose-dev.yml]: nice dot ;)

File: README.md (hunk 1 above)
Comment (wojwarych) [reply README.md]: Ok, will do.

File: docker/docker-compose-dev.yml (hunk 2 above)
Comment (wojwarych) [reply docker/docker-compose-dev.yml]: This is THAT DOT. :D

Comment (tdiak) [reply docker/docker-compose-dev.yml]: The best dot ever !

File: tests/README.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+
+On Windows Powershell  
+```PS
+.\run-server.bat /t
+```
+
+or cmd  
+```CMD
+run-server.bat /t
+```
```
Comment (tdiak) [inline tests/README.md]: Thanks for update :)

**Comment by wojwarych** [comment]
Wrong branch for pull ;)

**Pull request closed without merging**